src/Controls/src/Core/HandlerImpl/ContentView/ContentView.Impl.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public partial class ContentView : IContentView
+	{
+		object IContentView.Content => Content;
+		IView IContentView.PresentedContent => ((this as IControlTemplated).TemplateRoot as IView) ?? Content;
+
+		protected override void OnApplyTemplate()
+		{
+			Handler?.UpdateValue(nameof(Content));
+			base.OnApplyTemplate();
```

Comment:
Can we swap the order of these two?

I realize `base.OnApplyTemplate` is currently empty but it just feels like logically you'd want the `UpdateValue` call to happen after `OnApplyTemplate`